packages/nimble-components/build/generate-workers/tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "declaration": true,
+        "esModuleInterop": true,
+        "moduleResolution": "node",
+        "outDir": "dist/esm",
+        "strict": true,
+        "experimentalDecorators": true,
+        "target": "ES2020",
+        "module": "ES2020",
+        "allowJs": true,
+        "importHelpers": true,
+        "resolveJsonModule": true,
+        "sourceMap": true,
+        "lib": ["DOM", "ES2015", "ES2016.Array.Include", "ES2017.Object"],
+        "types": ["jasmine", "webpack-env"],
+        "importsNotUsedAsValues": "error",
+        "noImplicitOverride": true,
+        "noUncheckedIndexedAccess": true,
+        "noPropertyAccessFromIndexSignature": false
+    },
```

can you instead extend the nimble-components tsconfig?
```suggestion
    "extends": "../../tsconfig.json"
```